Triangulation_2/examples/Triangulation_2/constrained.cpp

```diff
@@ -22,10 +22,9 @@ main( )
 
   assert(cdt.is_valid());
   int count = 0;
-  for (CDT::Finite_edges_iterator eit = cdt.finite_edges_begin();
-       eit != cdt.finite_edges_end();
-       ++eit)
-    if (cdt.is_constrained(*eit)) ++count;
+  for (Edge e : cdt.finite_edges())
```

Comment:
This must grab by const& since the edge is not a handle

Reply:
done.